src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 40;
+pub const FORMAT_VERSION: u32 = 41;
```

Comment:
Another PR currently in the queue is making this change: https://github.com/rust-lang/rust/pull/138109#discussion_r1985883619

Once that lands, you should rebase ontop of that, and then bump again.